modules/installing-ocp-agent-inputs.adoc

```diff
@@ -77,7 +77,7 @@ pullSecret: '<pull_secret>' <5>
 sshKey: '<ssh_pub_key>' <6>
 EOF
 ----
-<1> Specify the system architecture, valid values are `amd64` and `arm64`.
+<1> Specify the system architecture, valid values are `amd64`, `arm64`, `ppc64le`, and `s390x`.
```

Comment:
Sorry to add last minute work to your plate, but the [Agent config reference](https://docs.openshift.com/container-platform/4.14/installing/installing_with_agent_based_installer/installation-config-parameters-agent.html) has been merged and is live, and that page has references to valid architectures too. Specifically, the descriptions of [compute.architecture](https://docs.openshift.com/container-platform/4.14/installing/installing_with_agent_based_installer/installation-config-parameters-agent.html#:~:text=compute%3A%0A%20%20architecture%3A-,Determines%20the%20instruction%20set%20architecture%20of%20the%20machines%20in%20the%20pool.%20Currently%2C%20clusters%20with%20varied%20architectures%20are%20not%20supported.%20All%20pools%20must%20specify%20the%20same%20architecture.%20Valid%20values%20are%20amd64%20and%20arm64.,-String) and [controlPlane.architecture](https://docs.openshift.com/container-platform/4.14/installing/installing_with_agent_based_installer/installation-config-parameters-agent.html#:~:text=controlPlane%3A%0A%20%20architecture%3A-,Determines%20the%20instruction%20set%20architecture%20of%20the%20machines%20in%20the%20pool.%20Currently%2C%20clusters%20with%20varied%20architectures%20are%20not%20supported.%20All%20pools%20must%20specify%20the%20same%20architecture.%20Valid%20values%20are%20amd64%20and%20arm64.,-String).


As of 2/9, you can find these Agent specific parameters in lines [492](https://github.com/openshift/openshift-docs/blob/main/modules/installation-configuration-parameters.adoc#L492) and [585](https://github.com/openshift/openshift-docs/blob/main/modules/installation-configuration-parameters.adoc#L585) of modules/installation-configuration-parameters.adoc